packages/api/src/hooks/useGetAccountTypes.ts

```diff
@@ -4,7 +4,7 @@ import useFetch from '../useFetch';
 /**
  * A custom hook to get available account types for a specific landing company.
  */
-const useAccountTypes = (landing_company?: string) => {
+const useGetAccountTypes = (landing_company?: string) => {
```

Comment:
@thisyahlen-deriv Keep it as before, The `get` in the name is redundant if we think of it as an entity.